.github/workflows/frontend-testing.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches: [master]
 
 jobs:
-  release-x86_64-linux-binaries:
+  upload-x86_64-linux-binaries:
```

Comment:
Why are we calling these `upload-*`? "release" seemed more correct, since we're also building.

Reply:
I did not want those to be confused with creating GitHub releases

Reply:
Sure, but it's creating a new build/release. I guess if you're worried about confusion with github release, you could call it `release-gcloud-*`

Reply:
Renamed accordingly